### PR TITLE
tidy ControlCatalog.cspoj

### PR DESCRIPTION
**Pull request by danwalmsley**

### CODE REVIEW

File: samples/ControlCatalog/ControlCatalog.csproj [hunk 1]
```diff
@@ -1,7 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
-    <EnableDefaultCompileItems>False</EnableDefaultCompileItems>
     <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
   </PropertyGroup>
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
```
Comment (grokys) [inline samples/ControlCatalog/ControlCatalog.csproj]: A lot of this stuff can be removed too. We should be basing this on our project template.

**Review by grokys — CHANGES_REQUESTED** [review]
While you're at it, could you trim the other unneccesary stuff too? :D

**Comment by danwalmsley** [comment]
Sure do you know which projects are affected?

**Comment by danwalmsley** [comment]
should be better now, do you know of any other project files that need a clean up?

**Review by grokys — APPROVED** [review]

**Comment by grokys** [comment]
The other samples could do with being converted to the new csproj format, but we can do that separately.

**Pull request merged**